integration-tests/tagsoup/src/test/resources/test.html

```diff
@@ -0,0 +1,26 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+    <p>Paragraph Contents</p>
```

Comment:
I'd say this document is too sane to see TagSoup at doing something really useful. Let's break it a bit  - e.g. let's at least remove the closing tag `</p>` to make it an invalid XML doc.

Reply:
TBH, I wasn't really worried about that. The main thing is that it runs and returns a document. I will add a syntax error, though.